src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.search--bar {
+  box-sizing: border-box;
```

Comment:
move `box-sizing: border-box;` to * selector

Reply:
Good!

Reply:
Please check, now should be ok.